docs/user-guide/cli-using-creating-profiles.md

```diff
@@ -244,3 +244,72 @@ In the following configuration, services are accessed through the API ML using c
     "autoStore": true
 }
 ```
+
+## Accessing services for multiple API ML instances
+
+In the following configuration, the profiles are structured to allow multiple instances of the API ML.
```

Comment:
I would also suggest adding a small note along the lines of:
"To authenticate to a specific API ML gateway from this config, you can run `zowe auth login apiml --base-profile lpar1` or `zowe auth login apiml --base-profile lpar2`.

Question for @t1m0thyj, would it make sense (or be more confusing) to have one of the `lpar` profiles set as the default base profile in this sample config?.